src/components/BnListbox/BnListbox.styles.cjs

```diff
@@ -40,6 +40,7 @@ module.exports = {
   // Separate class since it's teleported to body
   '.bn-listbox-options': {
     '@apply fixed max-h-60 w-full overflow-auto rounded-lg border border-gray-200 bg-white': {},
+    '@apply z-bn-listbox-options': {},
```

Comment:
If i understand,  this class is only to permit user to overide the default, but the default was not changed, isn't it?

Reply:
Oops, I forgot to add the default style to the commit.